### PR TITLE
New animals

### PR DESCRIPTION
**Pull request by Azaysol**
new animal update, lots of changes sending over to start review
- still pending sounds will send another update hopefully tomorrow with finalized sounds

### CODE REVIEW

**Review by alcatrazEscapee — CHANGES_REQUESTED** [review]
Need to fix merge conflicts. Need to also fix the pile of extra changes that have found their way into this PR:
- You should not be modifying `CodeStyle.xml`, nor anything in `build.gradle`, nor any of `generateResources.py`. None of those are relavant to this PR. In a similar manner there are many changes that need to be resolved with the main branch as they already exist. Please fix these as it will reduce the diff (and thus required review) signifigantly.

File: src/main/resources/assets/tfc/lang/en_us.lang [hunk 1]
```diff
@@ -792,7 +808,7 @@ config.tfc.animals.donkey=Donkey
 config.tfc.animals.mule=Mule
 config.tfc.animals.ocelot=Ocelot
 config.tfc.animals.wolf=Wolf
-config.tfc.animals.bear=Bear
+config.tfc.animals.bear=Grizzly Bear
```
Comment (HeartyYF) [inline src/main/resources/assets/tfc/lang/en_us.lang:811]: Only here, the key remains to be bear?

**Comment by DisasterMoo** [comment]
This commit: https://github.com/TerraFirmaCraft/TerraFirmaCraft/pull/1516/commits/9f592cc846de1b247c571706c7a1290a381dcbc9
is basically a copy-paste of changes made in the main repository. I recommend removing it and properly doing a merge in your branch.

Edit: Seems that there is some work on it. Well, this will be a little tricky, you have to move that work to another commit, so best doing it into another branch.

**Comment by Azaysol** [comment]
closing this request, to push new one from new branch

**Pull request closed without merging**